pvnet/models/base_model.py

```diff
@@ -414,6 +412,10 @@ def __init__(
         # save all validation results to array, so we can save these to weights n biases
         self.validation_epoch_results = []
 
+    def transfer_batch_to_device(self, batch, device, dataloader_idx):
```

Comment:
I feel like I might be missing something here, where is this function used?

Reply:
Its used behind the scenes by lightning, the same way that say the `training_step()` function is. We were previously using lightning's default batch transfer function, but this gives us more explicit control